lesson_15/tdd/tdd_app/src/test/java/com/codedifferently/lesson15/Lesson15Test.java

```diff
@@ -16,4 +17,74 @@ public void testGetGreeting() {
     // Act
     Lesson15.main(null);
   }
+
+  @Test
+  public void testEmployeeId_Exists() {
```

Comment:
The `Employee` related tests should be in a file named `EmployeeTest.java` to match the class they are testing. Similarly, `EmployeeManager` tests belong in an `EmployeeManagerTest.java` file.